titus-server-gateway/src/main/java/com/netflix/titus/gateway/eviction/EvictionResource.java

```diff
@@ -0,0 +1,140 @@
+package com.netflix.titus.gateway.eviction;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+
+import com.netflix.titus.api.eviction.service.EvictionException;
+import com.netflix.titus.api.model.Reference;
+import com.netflix.titus.api.model.Tier;
+import com.netflix.titus.common.util.StringExt;
+import com.netflix.titus.grpc.protogen.EvictionQuota;
+import com.netflix.titus.grpc.protogen.SystemDisruptionBudget;
+import com.netflix.titus.grpc.protogen.TaskTerminateResponse;
+import com.netflix.titus.runtime.connector.eviction.EvictionServiceClient;
+import com.netflix.titus.runtime.endpoint.common.rest.Responses;
+import com.netflix.titus.runtime.eviction.endpoint.grpc.GrpcEvictionModelConverters;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import rx.Observable;
+
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@Api(tags = "Eviction service")
+@Path("/v3/eviction")
+@Singleton
+public class EvictionResource {
+
+    private final EvictionServiceClient evictionServiceClient;
+
+    @Inject
+    public EvictionResource(EvictionServiceClient evictionServiceClient) {
+        this.evictionServiceClient = evictionServiceClient;
+    }
+
+    @ApiOperation("Return the global disruption budget")
+    @Path("budgets/global")
+    @GET
+    public SystemDisruptionBudget getGlobalDisruptionBudget() {
+        return Responses.fromSingleValueObservable(
+                evictionServiceClient
+                        .getDisruptionBudget(Reference.global())
+                        .map(GrpcEvictionModelConverters::toGrpcSystemDisruptionBudget)
+        );
+    }
+
+    @ApiOperation("Return a tier disruption budget")
+    @Path("budgets/tiers/{tier}")
+    @GET
+    public SystemDisruptionBudget getTierDisruptionBudget(@PathParam("tier") String tier) {
+        return Responses.fromSingleValueObservable(
+                evictionServiceClient
+                        .getDisruptionBudget(Reference.tier(StringExt.parseEnumIgnoreCase(tier, Tier.class)))
```

Comment:
maybe this is fine but passing in an invalid tier will result in a `Server Internal Error` here.

Reply:
I still did not define all error codes and exception mappings. I will do this later when the API stabilizes.